company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise2Test.java

```diff
@@ -45,10 +45,12 @@
     /**
      * Set up a {@link Predicate} that tests to see if a {@link Customer}'s city is "London".
      */
+    private static final Predicate<Customer> CUSTOMER_PREDICATE =  (Predicate<Customer>) customer -> "London".equals(customer.getCity());
```

Comment:
Hi @gonzra-bnym, please remove the cast to `Predicate<Customer`>. It should'd be necessary here.